src/app/pages/sign-up/shared/components/sign-up-first-step/sign-up-first-step.component.html

```diff
@@ -0,0 +1,55 @@
+<form class="form" [formGroup]="user" (ngSubmit)="onSubmit()">
+  <div class="fields">
+    <email-input formControlName="email"></email-input>
+    <div class="name">
+      <label class="field">
+        <p>First name</p>
```

Comment:
instead <p> should be lable, because there should be clickable just text, not a empty space around input

Reply:
Do you mean change 
```html
<label>
<p/>
<input/>
</label>
```
To
```html
<label for="firstname"/>
<input id="firstname"/>
```

If yes. How I should realize something like this with `password-input`?

Reply:
<div>
<label for="firstname"/>
<input id="firstname"/>
</div>

Reply:
<div>
<label for="firstname">
<input id="firstname"/>
img
<lable>
</div>
and you still can work with clickable img and clickable just input

Reply:
@valeriya-reva Remove label everywhere. Test label base functional (focus input on click) and it not work with some type of inputs or work bugly. Do this normal?